README.md

```diff
@@ -8,6 +8,17 @@ The scripts (e.g. startup_develop.sh) set up an environment for developing with
 `./startup_develop.sh down`: stops all container started with the script and remove container, volumes, etc.
 `./startup_develop.sh logs -f`: show logs of all running container metioned in the script file
 
-## Change version of container (env-files)
+## Change version or path of container (env-files)
 
-Version of docker container that aren't managed by the HPI-Schul-Cloud team could be inlcuded and update in `./envs/version.env` other docker tags should be added to the relatet env file (e.g. for develop us `./envs/develop.env`). Some of the container have also a default tag if nothing is set in envs. Changes for spezial need could also be added in a `.env` file inside the `./env` path and will not be commited. Please add credentials and secrets only in this file!
+Version of docker container that aren't managed by the HPI-Schul-Cloud team could be inlcuded and update in `./envs/default.env` other docker tags should be added to the relatet env file (e.g. for develop us `./envs/develop.env`). Some of the container have also a default tag if nothing is set in envs. **Envs for spezial need** could also be added in a `.env` file inside the `./envs` path and will not be commited. Please add credentials and secrets only in this file!
+
+## Structure
+
+**docker-compose** files are located in the `./compose-files` and base mostly on the `docker-compose.yml`, which starts the base setup of server, client and nuxt-client. Most of the others could change envs dependencies or links and set everything they need to run there services in combination wiht other containers (e.g. server).
+**env** files are localed in `./envs` and can be used in combination of the docker-compose files to set default values. Some of the container already have some default values if the env is not set. In some cases this is not used, because for the end-to-end tests they not allowed to set to trigger a pull. In **data** are files stored needed for the docker container (e.g. default setup data for the ldap server).
```

Comment:
```suggestion
**env** files are located in `./envs` and can be used in combination with the docker-compose files to set default values. Some of the containers already have some default values if the env is not set. In some cases this is not used, because for the end-to-end tests they not allowed to set to trigger a pull. In **data** files needed for the docker containers are stored (e.g. default setup data for the ldap server).
```

I don't understand what `In some cases this is not used, because for the end-to-end tests they not allowed to set to trigger a pull.` means. Can you explain it in another way?